pkg/config/config_test.go

```
@@ -0,0 +1,66 @@
+	f.WriteString("logs:\n")
+	f.WriteString("  file: /dev/stderr")
+	f.WriteString("  level: Info")
+	f.Close()
+	// get current working directory
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get current working directory: %v", err)
+	}
+	defer os.Chdir(cwd)
+	// change to tmp dir
+	os.Chdir(tmpDir)
```

golangci-lint (1.62.2)

Error return value of `os.Chdir` is not checked

(errcheck)